tests/Benchmark/EntryPoint.cpp

```diff
@@ -14,15 +14,52 @@
 //
 //******************************************************************************
 
-#include <windows.h>
 #include <TestFramework.h>
 #include "BenchmarkPublisher.h"
 #include <fstream>
 #include <algorithm>
 
 #ifdef WIN32
+#include <roapi.h>
+#include <wrl/client.h>
 #include <wrl/wrappers/corewrappers.h>
-using namespace Microsoft::WRL::Wrappers;
+#include <windows.storage.h>
+
+using namespace ABI::Windows::Storage;
+using namespace Microsoft::WRL;
+using namespace Windows::Foundation;
+
+Wrappers::HString GetAppDataPath() {
```

Comment:
It will but I didn't really want to introduce that dependency to Starboard since its a weird private function anyway. I can though if we feel thats nicer?